WebRoot/messages/ZmMsg.properties

```diff
@@ -4513,9 +4509,8 @@ twoStepAuthGenerateNewCodes = Generate New Codes
 twoStepAuthOneTimeCodesPrintTitle = Zimbra: One-time Codes
 twoFactorAuthCodeLabel = Code
 twoFactorAuthVerifyCode = Verify
-twoFactorAuthCancel = Cancel
 twoFactorAuthTrustDevice = Trust this device
-twoFactorAuthTrustComputer = Always trust this device
+twoFactorAuthTrustComputer = Trust this computer
```

Comment:
not sure computer is correct word to use here, it could be mobile device as well

Reply:
have used twoFactorAuthTrustDevice